cmd/root.go

```
@@ -123,8 +124,14 @@ func initCobraConfig() {
 	styles := boa.DefaultStyles()
 	b := boa.New(boa.WithStyles(styles))
 	oldUsageFunc := RootCmd.UsageFunc()
-	RootCmd.SetUsageFunc(b.UsageFunc)
-
+	RootCmd.SetUsageFunc(func(c *cobra.Command) error {
+		if c.Use == "atmos" {
+			b.UsageFunc(c)
```

golangci-lint (1.62.2)

Error return value of `b.UsageFunc` is not checked

(errcheck)